1-exercises/A-reading-list/script.js

```diff
@@ -1,6 +1,30 @@
+let content = document.querySelector('#content');
 function readingList(books) {
   // Write your code here...
-}
+  let unorderedList = document.createElement("ul"); // created unordered list tag
+  content.append(unorderedList);
+    
+  //iterate through the list of books
+   bookList = books.forEach(book => {
+    paraElement = document.createElement("p"); //p element
+    paraElement.innerHTML = (`${book.title} by ${book.author}`); // paragraph content
+    imageTag = document.createElement("img");
+    imageTag.src = book.bookCoverImage;
+    
+    let listElement = document.createElement("li"); // list
+    unorderedList.append(listElement);
+    listElement.append(paraElement);
+    listElement.append(imageTag);
+
+    //background colours
+      if (book.alreadyRead === true){
```

Comment:
Can remove the comparison to true as the predicate `book.alreadyRead` will produce the same result.